lib/models/config.ts

```diff
@@ -189,6 +194,39 @@ const getConfigModel = function (
 				| DeviceTypeJson.DeviceInitializationOptions
 			>(manifest.options, manifest.initialization?.options);
 		},
+
+		/**
+		 * @summary Get configuration variables schema for a device type
+		 * @name getConfigVarSchema
+		 * @public
+		 * @function
+		 * @memberof balena.models.config
+		 *
+		 * @param {String} deviceType - device type slug
+		 * @fulfil {Object[]} - configuration options
+		 * @returns {Promise}
+		 *
+		 * @example
+		 * balena.models.config.getConfigVarSchema('raspberry-pi').then(function(options) {
+		 * 	console.log(options);
+		 * });
+		 *
+		 * @example
+		 * balena.models.config.getConfigVarSchema('raspberry-pi', function(error, options) {
+		 * 	if (error) throw error;
+		 * 	console.log(options);
+		 * });
+		 */
+		getConfigVarSchema: async (
+			deviceType?: string,
+		): Promise<ConfigVarDefinition> => {
+			const { body } = await request.send({
+				method: 'GET',
+				url: `/config/vars${deviceType ? '?deviceType=${deviceType}' : ''}`,
```

Comment:
```suggestion
				url: `/config/vars${deviceType ? `?deviceType=${deviceType}` : ''}`,
```

Reply:
Actually, let's make this more explicit:
```suggestion
				url: `/config/vars${typeof deviceType === 'string' ? `?deviceType=${deviceType}` : ''}`,
```